tasks/darwin/disable-service.yml

```diff
@@ -3,7 +3,7 @@
   ansible.builtin.include_vars: "{{ role_path }}/vars/{{ ansible_os_family | lower }}.yml"
 
 - name: Get current UID
```

Comment:
In 7a0e6b5d4a676a9c471549d3a8db29303c50d688, it might be worth renaming the task, like in https://github.com/Ableton/ansible-role-prometheus-node-exporter/pull/96/commits/d3af7f62123d75e33691210094f12a7280ca437f
(This comment applies to `tasks/darwin/shutdown.yml` and `tasks/darwin/startup.yml` as well)

Reply:
Fixed in 779ea65.